packages/main/src/Avatar.js

```diff
@@ -226,6 +226,15 @@ const metadata = {
  * The shape can be circular or square. There are several predefined sizes, as well as an option to
  * set a custom size.
  *
+ * <br><br>
+ * <h3>Keyboard Handling</h3>
+ *
+ * <ul>
+ * <li>[SPACE, ENTER] - Fires the <code>ui5-click</code> event if the <code>interactive</code> property is set to true.</li>
+ * <li>[SHIFT] - If [SPACE] or [ENTER],[RETURN] is pressed, releases the <code>ui5-avatar</code> without triggering the click event.</li>
```

Comment:
If [SPACE] or [ENTER],[RETURN] is pressed, pressing [SHIFT] releases the <code>ui5-avatar</code> without triggering the click event.